### PR TITLE
Spelling fix

### PR DESCRIPTION
**Pull request by DenhamPreen**

### CODE REVIEW

File: docs/smart-contracts/plu-ts.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The tool is composed by the eDSL that allows you to create smart contracts and c
 
 ## Getting started
 
-Visit the [Get started with plu-ts](/docs/get-started/plu-ts) to install plu-ts unsing `npm` and setup your project.
+Visit the [Get started with plu-ts](/docs/get-started/plu-ts) to install plu-ts using `npm` and setup your project.
```
Comment (rphair) [inline docs/smart-contracts/plu-ts.md:17]:
```suggestion
Visit the [Get started with plu-ts](/docs/get-started/plu-ts) to install `plu-ts` using `npm` and set up your project.
```
... like this error (plus a breach of formatting convention) on the same line, for instance? 🤔

**Review by rphair — COMMENTED** [review]
I don't see a problem with merging this but the work involved is disproportionate to a single character change in the repository (it would be different if were in script or executable code).  @DenhamPreen what would you think of please spell checking some more Dev Portal material and adding some other spelling (and maybe grammar) corrections to this PR? 🤩🙏

**Comment by NotSimple** [comment]
This was fixed in https://github.com/cardano-foundation/developer-portal/pull/1243

**Comment by rphair** [comment]
well spotted @NotSimple ... originally reported issue fixed + suggestion above corrected now, so closing this.

**Pull request closed without merging**